src/test/pybind/test_cephfs.py

```diff
@@ -11,6 +11,11 @@ def setup_module():
     cephfs = libcephfs.LibCephFS(conffile='')
     cephfs.mount()
 
+def release_module():
+    global cephfs
+    cephfs = libcephfs.LibCephFS(conffile='')
+    cephfs.unmount()
```

Comment:
Surely you don't mean to construct `cephfs` and then call unmount on it immediately?